integration-tests/scenarios/grantDelegation.test.ts

```diff
@@ -159,16 +159,6 @@ describe("Delegation Scenario Tests", function () {
             }
         });
 
-        it("should fail to grant a duplicate delegation to a provider (DuplicateProvider)", async function () {
```

Comment:
Could we switch this to a positive test now? Aka an integration test of upsert with just a simple alter of the assert?

Reply:
What would be the benefit of this over the unit test that does that?

Reply:
Mostly just that extra end to end test that shows how it works. Not a blocker